docs/reference/api/metadata-stores/delta.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Delta Lake Metadata Store
+---
+
+# DeltaMetadataStore
+
+Delta Lake is the recommended object-store-backed metadata backend.
+It keeps one Delta table per feature, appends new metadata versions, and leans on Polars / Narwhals for provenance calculations.
+
+## Installation
+
+The backend relies on [`deltalake`](https://delta-io.github.io/delta-rs/python/)
+which ships with Metaxy’s `delta` dependency group:
+
+```bash
+uv sync --group delta
```

Comment:
This is obviously wrong for public docs, could you please proofread them?

Reply:
done; fixed